drivers/spi/spi_context.h

```diff
@@ -169,8 +169,8 @@ static inline void spi_context_cs_configure(struct spi_context *ctx)
 {
 	if (ctx->config->cs && ctx->config->cs->gpio_dev) {
 		gpio_pin_configure(ctx->config->cs->gpio_dev,
-				   ctx->config->cs->gpio_pin, GPIO_DIR_OUT);
-		gpio_pin_write(ctx->config->cs->gpio_dev,
+				   ctx->config->cs->gpio_pin, GPIO_OUTPUT);
+		gpio_pin_set(ctx->config->cs->gpio_dev,
 			       ctx->config->cs->gpio_pin,
```

Comment:
you will need to re-indent next parameter lines

Reply:
ack, done